### PR TITLE
IGNITE-15629 Baseline commands implemented

### PR DESCRIPTION
**Pull request by nizhikov**
Thank you for submitting the pull request to the Apache Ignite.

In order to streamline the review of the contribution 
we ask you to ensure the following steps have been taken:

### The Contribution Checklist
- [ ] There is a single JIRA ticket related to the pull request. 
- [ ] The web-link to the pull request is attached to the JIRA ticket.
- [ ] The JIRA ticket has the _Patch Available_ state.
- [ ] The pull request body describes changes that have been made. 
The description explains _WHAT_ and _WHY_ was made instead of _HOW_.
- [ ] The pull request title is treated as the final commit message. 
The following pattern must be used: `IGNITE-XXXX Change summary` where `XXXX` - number of JIRA issue.
- [ ] A reviewer has been mentioned through the JIRA comments 
(see [the Maintainers list](https://cwiki.apache.org/confluence/display/IGNITE/How+to+Contribute#HowtoContribute-ReviewProcessandMaintainers)) 
- [ ] The pull request has been checked by the Teamcity Bot and 
the `green visa` attached to the JIRA ticket (see [TC.Bot: Check PR](https://mtcga.gridgain.com/prs.html))

### Notes
- [How to Contribute](https://cwiki.apache.org/confluence/display/IGNITE/How+to+Contribute)
- [Coding abbreviation rules](https://cwiki.apache.org/confluence/display/IGNITE/Abbreviation+Rules)
- [Coding Guidelines](https://cwiki.apache.org/confluence/display/IGNITE/Coding+Guidelines)
- [Apache Ignite Teamcity Bot](https://cwiki.apache.org/confluence/display/IGNITE/Apache+Ignite+Teamcity+Bot)

If you need any help, please email dev@ignite.apache.org or ask anу advice on http://asf.slack.com _#ignite_ channel.

### CODE REVIEW

File: modules/codegen/src/main/java/org/apache/ignite/codegen/IgniteDataTransferObjectSerDesGenerator.java [hunk 1]
```diff
@@ -126,6 +130,12 @@ public class IgniteDataTransferObjectSerDesGenerator {
             false
         ));
 
+        TYPE_GENS.put(Long.class, F.t(
```
Comment (ivandasch) [inline modules/codegen/src/main/java/org/apache/ignite/codegen/IgniteDataTransferObjectSerDesGenerator.java:133]: Let's create the same for others boxed types?

File: modules/control-utility/src/main/java/org/apache/ignite/internal/commandline/DeclarativeCommandAdapter.java [hunk 2]
```diff
@@ -129,25 +128,27 @@ public DeclarativeCommandAdapter(org.apache.ignite.internal.management.api.Comma
...
-                    fld -> toArg.apply(fld, fld.getAnnotation(Argument.class).optional())
-                ).collect(Collectors.toList());
-
-                oneOfArgs.add(F.t(optionals, oneOfArg));
-
-                flds.forEach(fld -> namedArgs.add(toArg.apply(fld, true)));
-            }
+            positionalArgCb,
+            namedArgCb,
+            (argGrp, flds) -> flds.forEach(fld -> {
```
Comment (ivandasch) [inline modules/control-utility/src/main/java/org/apache/ignite/internal/commandline/DeclarativeCommandAdapter.java:146]: Is there any reason to leave this callback as is?

File: modules/core/src/main/java/org/apache/ignite/internal/management/AbstractCommandInvoker.java [hunk 3]
```diff
@@ -102,11 +103,20 @@ protected <A extends IgniteDataTransferObject> A argument(
     ) throws InstantiationException, IllegalAccessException {
         A arg = argCls.newInstance();
 
-        AtomicBoolean oneOfSet = new AtomicBoolean(false);
+        AtomicBoolean onlyOneOfGrp = new AtomicBoolean();
```
Comment (ivandasch) [inline modules/core/src/main/java/org/apache/ignite/internal/management/AbstractCommandInvoker.java:106]: May be it is better to create anonymous class that implements `Consumer` and has fields and set them approprieately?
Atomics here look quite weird, tbh

Comment (nizhikov) [reply modules/core/src/main/java/org/apache/ignite/internal/management/AbstractCommandInvoker.java:106]: Yes. Implemented

File: modules/control-utility/src/main/java/org/apache/ignite/internal/commandline/DeclarativeCommandAdapter.java (hunk 2 above)
Comment (nizhikov) [reply modules/control-utility/src/main/java/org/apache/ignite/internal/commandline/DeclarativeCommandAdapter.java:146]: Extracted to variable

**Review by ivandasch — APPROVED** [review]

**Pull request merged**